inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
I don't think we needed to split makeutype.c into BSD/GPL since we treat the info as "text and data strung-together to create something else", but to keep it simple, I'll switch is_LIGATURE.c to BSD license (treat it as new file replacing old - file while keeping same name).